service/routes.py

```diff
@@ -13,6 +13,15 @@
 from . import app
 
 
+############################################################
+# Health Endpoint
+############################################################
+@app.route("/health")
+def health():
+    """Health Status"""
```

Comment:
How about adding it to the logger as well?

Reply:
Changes made!